setup.py

```diff
@@ -91,12 +92,12 @@ def get_data_files():
 
 # Startup scripts - these use the mantidpython wrappers so we cannot
 # go through the entry_points mechanism
-scripts = ['scripts/start_mslice.py']
-if os.name == 'nt':
-    scripts.append('scripts/mslice.bat')
-else:
-    scripts.append('scripts/mslice')
-setup_args['scripts'] = scripts
+#scripts = ['scripts/start_mslice.py']
```

Comment:
Is this meant to be commented out, and could we delete it instead?

Reply:
That is a good question. If we are not installing MSlice standalone outside a conda environment anymore it can be just deleted. Otherwise I will probably have to remove the comments and only run this section if setup.py is not used within a conda environment.

Reply:
I've removed the commented out lines for now.